src/cli/commands/validator.ts

```diff
@@ -11,16 +11,18 @@ export async function handleValidatorCommand(args: ValidatorCommandArgs) {
   let config, configPath
   try {
     ;({ config, configPath } = await resolveConfig(args))
-    if (configPath != null) {
+    if (typeof configPath === 'string') {
```

Comment:
This is not necessary .. checking if it is defined at all is fine.
We could an `assert(typeof configPath === 'string')` after we determined it is non-null to alert users using the config wrong, but the initial check needs to stay the same.